tests/test_stream.js

```diff
@@ -245,6 +245,7 @@ test(
       res.setEncoding('utf8')
 
       let body = ''
+      t.equal(res.statusCode, 200)
```

Comment:
Could you update this test to use a non-default status code? It gives more confidence, I think.

```suggestion
      t.equal(res.statusCode, 201)
```